src/sidebar.js

```diff
@@ -29,7 +29,6 @@ module.exports = {
               ]
             },
             'start-building/native-mobile-app',
-            'guides/ory-cli-install-use',
```

Comment:
When moving documents please add the appropriate redirects as well:

https://github.com/ory/docs/blob/master/vercel.json

Reply:
Sure, will do. Wasn't aware of this procedure.

Reply:
That's done, see the vercel.js file.